### ISSUE TITLE
mongodbatlas_event_trigger is not updated if config_match is added

### ISSUE THREAD

**Issue by dachiarelli89** [mongodb/terraform-provider-mongodbatlas#1302]
### Terraform CLI and Terraform MongoDB Atlas Provider Version

```hcl
Terraform v1.4.2
on linux_amd64
+ provider registry.terraform.io/mongodb/mongodbatlas v1.10.0
```

### Terraform Configuration File 

```hcl
resource "mongodbatlas_event_trigger" "trigger" {
  project_id                  = data.terraform_remote_state.atlas.outputs.mongodb_project.project_id
  app_id                      = data.terraform_remote_state.atlas.outputs.application_id
  name                        = format("%s_%s", module.context.environment, "trigger")
  type                        = "DATABASE"
  disabled                    = false
  unordered                   = false
  config_operation_types      = ["INSERT", "DELETE", "REPLACE", "UPDATE"]
  config_providers            = ["anon-user"]
  config_database             = format("%s_%s", module.context.environment, "staging")
  config_collection           = "COLLECTION"
  config_service_id           = data.terraform_remote_state.atlas.outputs.config_service_id
  config_full_document        = false
  config_full_document_before = false
  config_schedule = "*"
  event_processors {
    aws_eventbridge {
      config_account_id = data.aws_caller_identity.current.account_id
      config_region     = var.aws_region
    }
  }
}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform apply` with script above
3. `terraform apply` adding this section

```hcl
  config_match    = <<-EOF
    {
      "$or": [
        {
          "operationType": {
            "$in": [
              "insert",
              "replace",
              "delete"
            ]
          }
        },
        {
          "$and": [
            {
              "$or": [
                {
                  "ADDRESS": {
                    "$exists": true
                  }
                },
                {
                  "STREET_NAME": {
                    "$exists": true
                  }
                },
                {
                  "ZIPCODE": {
                    "$exists": true
                  }
                },
                {
                  "CITY": {
                    "$exists": true
                  }
                },
              ]
            },
            {
              "operationType": "update"
            }
          ]
        }
      ]
    }
  EOF
```

### Expected Behavior
I expect that second run add a match expression filter on trigger.

### Actual Behavior
The trigger doens't change

**Comment by dachiarelli89** [comment]
Ok, there is a syntax error in the config_match
```hcl{
                  "CITY": {
                    "$exists": true
                  }
                },  <--- COMMA HERE
```

It could be useful have an error message in this case. Thank you

**Comment by andreaangiolillo** [comment]
Hello @dachiarelli89 
We released [v1.10.2](https://registry.terraform.io/providers/mongodb/mongodbatlas/1.10.2) with the fix to this bug. 

Thanks
